### ISSUE TITLE
markdownlint-configure-file is ignored when the comment spans multiple lines

### ISSUE THREAD

**Issue by LordAizen1** [rvben/rumdl#744]
`markdownlint-configure-file` only takes effect when the whole comment sits on one line. If it's split across lines, the config is silently ignored.

Works:

```markdown
<!-- markdownlint-configure-file { "MD013": { "line_length": 20 } } -->
```

Ignored:

```markdown
<!-- markdownlint-configure-file
{
  "MD013": { "line_length": 20 }
}
-->
```

With a 100 character line under each, the first reports MD013 at 20 and the second reports it at the default 80, so the block just doesn't register.

`parse_configure_file_comment` takes a single `line` and looks for `-->` in that same line, so a comment that hasn't closed yet doesn't parse and nothing else picks it up later.

Worth flagging how common the multi-line form is: it's what markdownlint's README shows for anything longer than a single setting, and 239 of the 407 test fixtures in the markdownlint repo use it.

I noticed this while diffing rumdl against markdownlint over those fixtures. It accounts for about 70% of the files where the two tools disagree, so it's probably hiding a fair amount of real parity in the comparison.

Happy to have a go at it if you want, but it needs the parser to buffer lines until the comment closes, and that seemed like a design call for you rather than something to put in a drive-by PR.

**Comment by rvben** [comment]
Fixed in v0.2.38 - `markdownlint-configure-file` comments spanning multiple lines are now honored (whole-document pre-pass). Thanks @LordAizen1.
